cmd/image-inspector.go

```diff
@@ -22,6 +22,7 @@ func main() {
 	flag.StringVar(&inspectorOptions.PasswordFile, "password-file", inspectorOptions.PasswordFile, "Location of a file that contains the password for authentication with the docker registry")
 	flag.StringVar(&inspectorOptions.ScanType, "scan-type", inspectorOptions.ScanType, fmt.Sprintf("The type of the scan to be done on the inspected image. Available scan types are: %v", iicmd.ScanOptions))
 	flag.StringVar(&inspectorOptions.ScanResultsDir, "scan-results-dir", inspectorOptions.ScanResultsDir, "The directory that will contain the results of the scan")
+	flag.BoolVar(&inspectorOptions.Html, "html", inspectorOptions.Html, "Generate an OpenScap HTML report in addition to the ARF formatted report")
```

Comment:
> A configuration file might be useful. The only question I have is when CloudForms invokes the image-inspector, how would we feed the config file to the inspector?

Assuming it is running as a pod it could be

1.  built into the image
1.  added as a mount

Just something to consider if this is going to be a common maintenance issue.